src/events/ready.js

```diff
@@ -49,5 +49,14 @@ module.exports = {
 
             formChannel.send({ embeds: [registerServerEmbed, requestDevRoleEmbed], components: [row] });
         }
+
+        // se crashar ou reiniciar no meio de uma atualização, ele vai refazer essa atualização já
+        // que a ela não foi finalizada devidamente
+        const constantsModel = require('../models/constants');
+        const constants = await constantsModel.getConstants();
+        if (constants.updatingGuildsChannel || constants.scheduledUpdate) {
+            await constantsModel.updateConstants({ updatingGuildsChannel: false, scheduledUpdate: false });
+            client.emit('updateGuilds');
```

Comment:
explicacao no primeiro comentario em src/events/updateGuilds.js
```suggestion
            client.emit('updateGuilds', false);
```